src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/DynamicFlags.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.ExpressionEvaluator
+{
+    /// <remarks>
+    /// WARNING: Mutable struct.
+    /// </remarks>
+    internal struct DynamicFlags
```

Comment:
As we discussed offline, it is not building anything.  I started out with a class, but switched to a struct so that the default instance would act as the empty instance without special-casing it everywhere.